features/support/commit_helpers.rb

```diff
@@ -132,11 +146,15 @@ def recent_commit_shas count
 def verify_commits commits_array
   normalize_commit_data commits_array
 
-  expected_commits = commits_array.map do |commit_data|
-    normalize_expected_commit_data commit_data
-  end.flatten
-
+  expected_commits = normalize_expected_commits_array
```

Comment:
Ah, sorry I had a cold last week, and was to fuzzy-minded to catch it and fix this earlier. Thanks!